src/Eshopworld.WorkerProcess/WorkerLease.cs

```diff
@@ -75,8 +75,6 @@ internal async Task<TimeSpan> LeaseAsync()
         {
             if (CheckLeaseExpired(CurrentLease))
             {
-                await _leaseAllocator.ReleaseLeaseAsync(CurrentLease).ConfigureAwait(false);
```

Comment:
Do we use that method anywhere else? Should we remove it from the store as well?

Reply:
We could, but since the method is public and could be in use by any of the users of the package, I didn't remove it. Maybe if we add a way to expire a lease manually then we can remove that as well.

Reply:
No, I can't imagine why it should be used by anyone outside of the package - can you at least mark it as obsolete, please?